WCON_format.md

```diff
@@ -226,9 +234,51 @@ For features that describe an entire plate rather than properties of individual
 
 This example presents only a small number of possibilities for how a custom reader could store information within a WCON file.  The driving principle is that the custom readers and writers can do whatever they need to, so long as it is valid JSON, and any units that ought to be automatically converted are specified in the `"units"` block.
 
-## Common additions handled in accompanying code
+## Common additions supported by the Tracker Commons project
+
+It is expected that users will take advantage of the flexibility of the WCON specification to create derivatives.  The following examples are handled by the full-featured readers and writers in the Tracker Commons project.  We recommend that general-purpose code written in other languages also handle these cases.
 
-It is expected that users will take advantage of the flexibility of the WCON specification to create derivatives.  The following examples are handled by the accompanying code.  We recommend that general-purpose code written in other languages also handle these cases.
+### Local constants
+
+Some values vary by animal but not from time to time.  These may be specified as _local constants_ even when the time is arrayed in order to save space: instead of repeating the value many times in an array, the value is only given once.  Entries that must be read by a minimal reader may _not_ be local constants in order to simplify interpretation.  Optional values that may be local constants are specified below.
```

Comment:
Interesting.  So we have disallowed single-valued data items for 'ox', 'oy' but not for custom additions.  OK, fair enough.